unstructured/documents/html.py

```diff
@@ -60,8 +60,12 @@ def load(cls, opts: HtmlPartitionerOptions) -> HTMLDocument:
 
     @lazyproperty
     def elements(self) -> list[Element]:
-        """Gets all elements from pages in sequential order."""
-        return [el for page in self.pages for el in page.elements]
+        """All "regular" elements (e.g. Title, NarrativeText, etc.) parsed from document.
+
+        Elements appear in document order. Elements that appeared in a header or footer are filtered
+        out.
```

Comment:
Are we adding back header and footer elements later?

Reply:
Yep, good catch. Not sure where that came from but it's gone now :)